cmd/runtimetest/main.go

```diff
@@ -370,6 +371,52 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
+func getActualCgroupPath() (string, error) {
+	f, err := os.Open("/proc/self/cgroup")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return "", err
+		}
+
+		text := s.Text()
+		parts := strings.Split(text, ":")
+		return parts[2], nil
+	}
+	return "", fmt.Errorf("can not get cgroup path")
+}
+
+func validateCgroupsPath(spec *rspec.Spec) error {
+	logrus.Debugf("validating cgroupsPath")
+	expectedPath := spec.Linux.CgroupsPath
+	if expectedPath == nil {
+		return nil
+	}
+	*expectedPath = strings.Replace(*expectedPath, ":", "/", -1)
```

Comment:
There is actually not mention of colon->slash. But runc do like this. I we leave this out, I guess most of runtime may fail.